src/Microsoft.Framework.Runtime.Interfaces/IServiceProviderLocator.cs

```diff
@@ -5,8 +5,14 @@
 
 namespace Microsoft.Framework.Runtime.Infrastructure
 {
+    /// <summary>
+    /// The <see cref="IServiceProviderLocator"/> provides access to the current <see cref="IServiceProvider"/>.
+    /// </summary>
     public interface IServiceProviderLocator
     {
+        /// <summary>
+        /// Gets or sets the <see cref="IServiceProvider"/>.
+        /// </summary>
```

Comment:
/// &lt;summary>
/// Retrieves the active @victorhurdugaci 
/// &lt;/summary>
/// &lt;remarks>
/// This method retrieves the @victorhurdugaci that is currently active. Warning, @victorhurdugaci is a troll
/// &lt;/remarks>
/// &lt;value>
/// The active @victorhurdugaci
/// &lt;/value>
/// &lt;returns>
/// The @victorhurdugaci that is currently active
/// &lt;/returns>